pkg/storage/filesystem.go

```diff
@@ -20,24 +20,25 @@ type Mountpoint struct {
 	FsType     string
 }
 
+func GetSupportedFilesystems() (fstypes[]string, err error) {
+	return internalGetFilesystems("/proc/filesystems")
+}
+
 // GetSupportedFilesystems returns the supported file systems for block devices,
-func GetSupportedFilesystems() ([]string, error) {
-	fd, err := os.Open("/proc/filesystems")
-	if err != nil {
-		return nil, err
+func internalGetFilesystems(file string) (fstypes []string, err error) {
+	var bytes []byte
+	if bytes, err = ioutil.ReadFile(file); err != nil{
+		return nil, fmt.Errorf("Failed to read /proc/filesystems: %v", err)
```

Comment:
Since the file that you're reading is not necessarily `/proc/filesystems` anymore, lets change this error to be `fmt.Errorf("Failed to read %s: %v", file, err)`, so that it prints the given file name